data-raw/internal-data.R

```diff
@@ -5,6 +5,68 @@ passphrase <- 'somepassphrase'
 # credentials_name <- "credentials.sqlite"
 
 
+# Overall descriptive test for contact sub-tab under About tab.
```

Comment:
I actually decided it's a little tedious to store this info inside of the `sysdata.rda` object, and much easier to just change it directly in the UI. Also, got rid of calls to `uiOutput()` since we aren't displaying anything dynamically.